images/base/Dockerfile

```diff
@@ -52,8 +52,8 @@ RUN chmod +x /usr/local/bin/clean-install
 RUN clean-install \
       apt-transport-https ca-certificates curl software-properties-common gnupg2 lsb-release \
       systemd systemd-sysv libsystemd0 \
-      iptables iproute2 ethtool socat util-linux mount ebtables udev kmod aufs-tools \
```

Comment:
hmm we explicitly need these though, what's the downside to installing them?
we may change the base image again in the future. 🤔

Reply:
Duplicate installations only add time, but because they already exist, there is nothing else going on.

Reply:
For the current situation, the base images are built ahead of time and it seems that there will be no problem. It’s just my habit of building an image that will remove unwanted things.

Reply:
👍 

It shouldn't add much time though? Ot just needs to look up that they are installed and then skip them. 

The base image is intended to only be built when we change it / upgrade a package.  Ideally not very often. I might be missing a use case though 🙃

Reply:
yes. I will close it.

Reply:
thanks again anyhow! always appreciate PRs :-)